src/wasm/literal.cpp

```diff
@@ -218,21 +217,7 @@ void Literal::getBits(uint8_t (&buf)[16]) const {
     case Type::v128:
       memcpy(buf, &v128, sizeof(v128));
       break;
-    // TODO: investigate changing bits returned for reference types. currently,
-    // `null` values and even non-`null` functions return all zeroes, but only
-    // to avoid introducing a functional change.
-    case Type::funcref:
-      break;
-    case Type::externref:
-    case Type::exnref:
-    case Type::anyref:
-    case Type::eqref:
-      assert(isNull() && "unexpected non-null reference type literal");
-      break;
-    case Type::i31ref:
-      WASM_UNREACHABLE("TODO: i31ref");
-    case Type::none:
-    case Type::unreachable:
+    default:
```

Comment:
We generally try to avoid using `default:` in switches over the basic types, since avoiding it makes the compiler tell us about all the places we need to update when we add a new basic type. On the other hand, you're the one who has been adding the most new types recently, so if this doesn't seem very useful to you, we could revisit that policy.

Reply:
Thinking here was that the function is really only valid for the five basic value types after the change and I don't anticipate these to be extended anytime soon. Perhaps `f128` or larger vectors, hmm. Not feeling strongly.